docs/new_customdoxygen.css

```diff
@@ -1997,3 +1993,279 @@ background-color: #f7f7f7;
 
 /*  End */
 
+/* @Footer */
+/* NewFooter0 */
+.NewFooter0{
+font-size:13px;
+font-family:proxima-nova,"Proxima Nova",sans-serif;
+color:#ffffff;
+-webkit-letter-spacing:0.56px;
+-moz-letter-spacing:0.56px;
+-ms-letter-spacing:0.56px;
+letter-spacing:0.56px;
```

Comment:
Write in better code-style - indentation and spaces after `:`. Hard to read now.

Reply:
This is not Alexander's business.

Reply:
@e-schepachev But he can do it. It can be done even by computer program - "beautifier" 😆

Reply:
e-schepachev boostmultifruit
Data in the file new_customdoxygen.css is quite large and was generated automatically. But I'll try to format the data of entire file as soon as possible.